unlock-app/src/components/interface/checkout/Checkout.tsx

```diff
@@ -108,12 +115,24 @@ export const Checkout = ({
   const [existingKeys, setHasKey] = useReducer(keysReducer, {})
   const [selectedLock, selectLock] = useState<any>(null)
   const [savedMetadata, setSavedMetadata] = useState<any>(false)
+  const [storedLoginEmail, setStoredLoginEmail] = useState<string>('')
```

Comment:
Maybe we could move that inside the `useAutoLogin` hook?

Reply:
I defined the `storedLoginEmail` state in this component so we don't lose the value, we use the hook only to get the value